lib/events/types.go

```diff
@@ -60,6 +60,7 @@ type (
 	SessionStart                    = events.SessionStart
 	SessionJoin                     = events.SessionJoin
 	SessionPrint                    = events.SessionPrint
+	SessionUpload                   = events.SessionUpload
```

Comment:
This is not necessary - since this is new code, you can use the `api/types/events` package directly.